README.md

```diff
@@ -66,6 +65,21 @@ s = shell.quote(p)
 
 ## Writing a new module
 
+The criteria for adding a new function or module to this repository are:
+
+1. Is it widely needed? The new code must solve a problem that occurs often during the development of Bazel build rules. It is not sufficient that the new code is merely useful.  Candidate code should generally have been proven to be indispensible in the repository in which it was developed, or across several projects.
```

Comment:
"Indispensible" strikes me as too strong of wording. For example, the build settings in rules/common_settings.bzl were created at the inception of build settings because we knew it would prevent the implementation of many extremely-similar build settings across projects and allowed us to get simple ones out there that used best practices.

So IMO another reasonable reason to add content is because it's going to steer users towards a more standardized way to use features which is known to work and will save them implementation time. WDYT?

Reply:
Rephrased.